website/server/models/user/schema.js

```diff
@@ -521,6 +521,7 @@ let schema = new Schema({
     return {};
   }},
   pushDevices: [PushDeviceSchema],
+  ABtest: {type: String},
```

Comment:
Shoot, that makes it show up `undefined` during the integration test. Is the test wrong, or the way I found to make the property private?